README.md

```diff
@@ -1,4 +1,4 @@
-# Gonum [![Build Status](https://travis-ci.org/gonum/gonum.svg?branch=master)](https://travis-ci.org/gonum/gonum) [![Coverage Status](https://coveralls.io/repos/gonum/gonum/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/gonum?branch=master) [![GoDoc](https://godoc.org/gonum.org/v1/gonum?status.svg)](https://godoc.org/gonum.org/v1/gonum)
+# Gonum [![Build Status](https://travis-ci.org/gonum/gonum.svg?branch=master)](https://travis-ci.org/gonum/gonum) [![Coverage Status](https://coveralls.io/repos/gonum/gonum/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/gonum?branch=master) [![GoDoc](https://godoc.org/gonum.org/v1/gonum?status.svg)](https://godoc.org/gonum.org/v1/gonum) [![Go Report Card](https://goreportcard.com/badge/github.com/gonum/gonum)](https://goreportcard.com/report/github.com/gonum/gonum)
```

Comment:
Ah, that explains why when I ran it on the "go get" path (what the documentation says), I got 0% for all the checks. In other news, the number of false positives is annoying (Statuser is not a misspelling of Stature, it's actually following Go convention)